api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -524,6 +525,25 @@ export class ScenariosController {
     return AsyncJobDto.forScenario().asJsonApiMetadata();
   }
 
+  @ApiTags(asyncJobTag)
+  @ApiOkResponse()
+  @Delete(':id/planning-units/status/:kind')
+  async clearPlanningUnitsStatus(
+    @Param('id', ParseUUIDPipe) id: string,
+    @Param('kind') kind: string,
```

Comment:
you could (should!) enforce that `kind` is one of the vetted `LockStatus` enum values. If it's not any of these, we can fail fast and return a 400 directly here (ideally via class-validator)